modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -588,11 +658,11 @@ use {{invokerPackage}}\ObjectSerializer;
                 $httpBody = new MultipartStream($multipartContents);
 
             } elseif ($headers['Content-Type'] === 'application/json') {
-                $httpBody = \GuzzleHttp\json_encode($formParams);
+                $httpBody = json_encode($formParams);
 
             } else {
                 // for HTTP post (form)
-                $httpBody = \GuzzleHttp\Psr7\build_query($formParams);
+                $httpBody = Query::build($formParams);
```

Comment:
Is this still required?

Reply:
guzzlehttp/psr7 advices to use a different method from the same package instead
> @deprecated build_query will be removed in guzzlehttp/psr7:2.0. Use Query::build instead.